models/AmazonS3.cfc

```diff
@@ -400,12 +427,14 @@ component accessors="true" singleton {
 		string maxKeys             = "",
 		string delimiter           = variables.defaultDelimiter
 	) {
-		requireBucketName( arguments.bucketName );
-		var parameters = {};
+        requireBucketName( arguments.bucketName );
+        var parameters = {
+			"list-type" : 2
```

Comment:
Can you put a link here in a comment to this API Doc. Would be great to have it referenced

Reply:
I've placed this in the `@see` in the doc blocks